app/editor/src/features/admin/ingests/configurations/AudioStream.tsx

```diff
@@ -58,6 +58,7 @@ export const AudioStream: React.FC = (props) => {
         label="Other Arguments"
         name="configuration.otherArgs"
         tooltip="Any other arguments to pass to the command"
+        placeholder="-acodec mp3 -ab 257k"
```

Comment:
Based on blogs this seems to be a good idea... I have no idea really.  I'm just testing if it's better, but so far it seems to work the same as without it.